### ISSUE TITLE
Entering wrong password on Panel Login screen gives says "Invalid argument - in method -"

### ISSUE THREAD

**Issue by neildaniels** [getkirby/kirby#3006]
**Describe the bug**  
The error reporting for the Panel login does not give a reasonable error message when entering in the wrong password for an actual Panel user.

**To Reproduce**  
1. Enable **Debug Mode**
2. Log out of Panel
3. Enter email address of extant Panel user
4. Enter incorrect password
5. Click "Login"

**Expected behavior**  
Says something like "wrong password"

**Screenshots**  
<img width="506" alt="Screen Shot 2020-12-14 at 3 30 48 PM" src="https://user-images.githubusercontent.com/417182/102148470-583a7100-3e21-11eb-8155-47ce7783b38c.png">

**Kirby Version**  
`3.5.0-rc.7`

**Console output**  
```
[Error] Object

code: 400

details: [] (0)

exception: "Kirby\\Exception\\InvalidArgumentException"

file: "User.php"

key: "error.user.password.notSame"

line: 945

message: "Invalid argument \"-\" in method \"-\""

route: "auth/login"

status: "error"

Object Prototype

	onError (app.js:1:22156)
	(anonymous function) (app.js:1:17805)
	asyncFunctionResume
	(anonymous function)
	promiseReactionJobWithoutPromise
	promiseReactionJob
```

**Desktop (please complete the following information):**  
 - OS: macOS
 - Browser: Safari 14.0.2 (16610.3.7.1.9)

**Comment by afbora** [comment]
I've tested on 3.5.0-rc.7 and couldn't reproduce it. 
Indeed `error.user.password.notSame` translation key exists for all languages.

**Comment by neildaniels** [comment]
Apparently, I can only reproduce this when Kirby is in **Debug Mode**. (Updated original issue with this step.)

@afbora Can you reproduce with Debug Mode enabled?

**Comment by afbora** [comment]
@neildaniels No, I always run my tests in debug mode. Are you getting the same error on Starterkit + RC.7?

**Comment by neildaniels** [comment]
I finally narrowed down a (hopefully) full reproduction scenario:

The issue occurs when:
- Kirby is in multi-language mode
- The default language has a code that doesn't match Kirby's set of built-in translations

In my simplified test, I took the "3.5.0 Starterkit" and changed the `config.php` to:
```php
return [
    'debug' => true,
    'languages' => true,
];
```

- Create an initial Panel user
- In the Panel, create the first language with the following details:
  - Name: US English
  - Code: `en-us`
  - Locale: `en_US`
<img width="607" alt="Screen Shot 2020-12-15 at 12 09 20 PM" src="https://user-images.githubusercontent.com/417182/102267803-47e0cf80-3ecf-11eb-99f7-df2e7c32d977.png">

- Log Out of Panel
- Try to log in with the wrong password

- - -

I would have expected that it would use the "fallback" of `en` in the I18n class, which is seemingly what `3.4.5` did.

I didn't see any obvious, recent changes in the [Exception](https://github.com/getkirby/kirby/blob/3.5.0/src/Exception/Exception.php) or [I18n](https://github.com/getkirby/kirby/blob/3.5.0/src/Toolkit/I18n.php) classes that would seem to cause this behavior.

**Comment by afbora** [comment]
Yes I can reproduce the issue now with `en-us` code.

**Comment by lukasbestle** [comment]
This is because our [locale fallback handler](https://github.com/getkirby/kirby/blob/3.5.0/src/Cms/AppTranslations.php#L57-L63) always uses the language code of the default language as the fallback translation locale. When setting a language code that Kirby's translations don't know about (like `en-us`), Kirby can therefore not find a translation.

The code I linked to above was already in Kirby 3.0.0, but maybe this has now surfaced because of the changes to the login handling in Kirby 3.5.0. The general issue was there before that.

I have fixed this in #3021 by adding another fallback to `en` that will be used as a last resort.
